sources/controllers/node_controller.rb

```diff
@@ -11,28 +9,35 @@ class NodeController < Sinatra::Base
   end
 
   # html view, sends stuff to /node as POST
-  get '/new' do
-    @node = Node.new
-    haml :node
-  end
-
-  post '/' do
-    Node.new(params[:node])
-  end
+  get '/new' do             #
+    @node = Node.new        # dafuq is this
+    haml :node              #
+  end                       #
 
-  delete '/:id' do
-    Node.delete(params[:id])
-  end
+  post '/' do               #
+    Node.new(params[:node]) # dafuq is that
```

Comment:
params is a hash, the symbol ```:node``` contains another hash with all attributes for a new domain (this is C&P from the sinatra quickstart guide written by the sinatra founder)